server/portal/apps/workspace/unit_test.py

```diff
@@ -64,8 +64,8 @@ def test_apps_list(self):
 
         # need to do a return_value on the mock_client because
         # the calling signature is something like client = Agave(**kwargs).apps.list()
-        self.mock_client.apps.list.return_value = apps
```

Comment:
/worksapce/unit_test.py is really testing the api (i.e. workspace/api) so we should move its contents there.  i got a local note for that.

in general, it would be nice for the workspace/api/views.py should be split up and workspace/api/views_unit_test.py but that would add up more confusion at the moment so DEFINITELY not going to touch that.